external/CMakeLists.txt

```
@@ -30,19 +30,7 @@ if (BUILD_WITH_BLASFEO)
     add_subdirectory(blasfeo)
 
 else()
-    set(BLASFEO_INSTALL_DIR "/opt/blasfeo" CACHE PATH "Path to blasfeo installation directory")
-    # find_library(blasfeo NAMES blasfeo PATHS ${BLASFEO_INSTALL_DIR}/lib)
-
-    find_path(BLASFEO_INCLUDE_DIR NAMES "blasfeo.h" PATHS ${BLASFEO_INSTALL_DIR}/include)
-    find_library(BLASFEO_LIBRARY NAMES blasfeo HINTS ${BLASFEO_INSTALL_DIR}/lib)
-
-    # Create an imported static target for blasfeo
-    add_library(blasfeo STATIC IMPORTED GLOBAL)
-    # Set the path to the library file
-    set_target_properties(blasfeo PROPERTIES IMPORTED_LOCATION ${BLASFEO_LIBRARY})
-
-    # Set the path to the include directory
-    set_target_properties(blasfeo PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${BLASFEO_INCLUDE_DIR})
+    find_package(blasfeo REQUIRED)
```

```suggestion
    find_package(blasfeo REQUIRED CONFIG)
```

If the goal is to use blasfeoConfig.cmake, we can make that explicit

---

Personally I am not a big fan of this, but this is mostly personal taste/bikeshedding. If someone wants to use `fatrop` in a bigger CMake project via `add_subdirectory`/`FetchContent`, they may want to override the logic happening in `find_package(blasfeo REQUIRED)` with something custom, for whatever reason. This can be done by adding a `Findblasfeo.cmake` in the CMAKE_MODULE_PATH, but this will not work if downstream projects manually add `CONFIG` in their `find_package` calls. However, I do not feel strongly about this, so if mantainers perfer to have it it is ok for me to have it.